insights/parsers/mysqladmin.py

```diff
@@ -15,6 +15,38 @@
 from insights.specs import Specs
 
 
+@parser(Specs.mysqladmin_status)
+class MysqladminStatus(LegacyItemAccess, CommandParser):
+    """
+    Module for parsing the output of the ``mysqladmin status`` command.
+    Typical output looks like::
+        Uptime: 1103965 Threads: 1820 Questions: 44778091 Slow queries: 0 Opens: 1919 Flush tables: 1 Open tables: 592 Queries per second avg: 40.561
+    Example:
+        >>> "Uptime" in mysqlstat
+        True
+        >>> mysqlstat['Threads']
+        '1820'
+        >>> mysqlstat['Flush_tables'] == '1'
+        True
+    """
```

Comment:
You have some errors in your docstring.  Please follow the instructions here to test and confirm that your documentation is accurate, https://insights-core.readthedocs.io/en/latest/docs_guidelines.html